test/unit/typescript/test-element.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Test is failing in firefox: https://travis-ci.org/Polymer/polymer/builds/159791491#L1052

`The module "http://localhost:2000/components/polymer/test/unit/typescript/test-element.js" has not been loaded`